subsystem_objects/Spacecraft_AODCS

```diff
@@ -0,0 +1,48 @@
+{
+    "@id": "http://ontology.projectchronos.eu/subsystems/Spacecraft_AODCS",
+    "@type" :"http://www.w3.org/2002/07/owl#Class",
+    "rdfs:subClassOf" : [
+        { "@id": "http://ontology.projectchronos.eu/spacecraft/Subsystem_Spacecraft" },
```

Comment:
Guys none of these links are working. 
@Mec-iS ^^ can you provide the definitions we need or maybe start this server again? 
Thanks a lot!

Reply:
yes these are not working we need to change .